README.md

```diff
@@ -15,17 +18,24 @@ If you experience any issues with the older versions, you are most welcome to co
 
 ## Status
 
-Both the OPTIMADE specification and this repository are **under development**.
+Whilst v1.0 of the OPTIMADE specification has been released, this repository is **under development**.
+Outstanding features required for compliance with OPTIMADE v1.0 can be tracked with the OPTIMADE v1.0 label on [GitHub](https://github.com/Materials-Consortia/optimade-python-tools/issues?q=is%3Aopen+is%3Aissue+label%3A%22OPTIMADE+v1.0%22), which can be further filtered by backend.
+The package changelog can be found in [CHANGELOG.md](changelog.md)
 
-The latest stable version can be obtained from [PyPI](https://pypi.org/project/optimade) `pip install optimade` or by cloning the master branch of this repository `git clone git@github.com:Materials-Consortia/optimade-python-tools`.
+## Documentation
+
+This document, guides, and the full module API documentation can be found at [https://optimade.org/optimade-python-tools](https://optimade.org/optimade-python-tools).
 
 ## Installation
 
-Installation instructions, for both the index meta-database, and for the main API can be found in [the documentation](https://www.optimade.org/optimade-python-tools/install/).
+Installation instructions, for both the index meta-database, and for the main API can be found in [INSTALL.md](install.md).
```

Comment:
Same comment as above for changelog.
Note, for both of these the link was wrong, I think?
```suggestion
Installation instructions, for both the index meta-database, and for the main API can be found in [the installation documentation](INSTALL.md).
```